styles/github/catppuccin.user.css

```diff
@@ -325,7 +325,7 @@
       background: var(--color-btn-danger-bg);
     }
 
-    /* Create PR button & New branch button & Frok repo button & Commit Changes button*/
+    /* Create PR button & New branch button & Fork repo button & Commit Changes button */
```

Comment:
How about `/* New PR, new branch, fork repository, and commit changes buttons */`?